### DISCUSSION TITLE
How to use PHP CS Fixer to make each method call on its own line in a chain of method calls?

### DISCUSSION THREAD

**Discussion by ulin-evgeny** [PHP-CS-Fixer/PHP-CS-Fixer#7854] [Q&A]
Code before processing:

    $this->create()->content()
    ->asd($result)
        ->json($result);

Code after processing:

    $this
        ->create()
        ->content()
        ->asd($result)
        ->json($result);

This option is also suitable, but the previous one is preferable:

    $this->create()
        ->content()
        ->asd($result)
        ->json($result);

**Comment by Wirone** [comment]
I believe we don't have such rule currently. We have `method_chaining_indentation` which handles indentation in chained calls that already are multi-line, but we don't have rule that would enforce multi-line chained calls.
